src/api/db/migrate/20190204162828_create_watch_item.rb

```diff
@@ -0,0 +1,11 @@
+class CreateWatchItem < ActiveRecord::Migration[5.2]
+  def change
+    create_table :watch_items, id: :integer do |t|
+      t.references :user, index: true
```

Comment:
this should be `int` and not `big_int` (as `id` in `user` is int). Have you use the generator to create this migration? I think it should take care of it... not sure though... :thinking: